### PR TITLE
Document the policy for repositories under the Python org

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

File: devcycle.rst [hunk 1]
```diff
@@ -215,7 +215,19 @@ Repository Administration
...
+
+Organization Repository Policy
+------------------------------
+
+Within the `Python organization <https://github.com/python/>`_, repositories are expected to fall within three general categories:
+
+1. The refence implementation of Python and related repositories (i.e. `CPython <https://github.com/python/cpython>`_)
+2. Reference implementations of PEPs (e.g. `mypy <https://github.com/python/mypy>`_)
+3. Tooling and support around CPython and the language (e.g. `python.org repository <https://github.com/python/pythondotorg>`_)
+
```
Comment (ewdurbin) [inline devcycle.rst]: I'd like to see explicit inclusion of generalized PSF and PSF Infrastructure related projects.

**Comment by brettcannon** [comment]
@ewdurbin PTAL

File: devcycle.rst [hunk 2]
```diff
@@ -215,7 +215,20 @@ Repository Administration
...
 The source code is currently hosted on `GitHub
-<https://github.com/python/cpython>`_.
+<https://github.com/python/cpython>`_ in the `Python organization <https://github.com/python/>`_.
+
+Organization Repository Policy
+------------------------------
+
+Within the `Python organization <https://github.com/python/>`_, repositories are expected to fall within three general categories:
+
+1. The refence implementation of Python and related repositories (i.e. `CPython <https://github.com/python/cpython>`_)
```
Comment (willingc) [inline devcycle.rst]:
```suggestion
1. The reference implementation of Python and related repositories (i.e. `CPython <https://github.com/python/cpython>`_)
```

File: devcycle.rst [hunk 3]
```diff
@@ -215,7 +215,20 @@ Repository Administration
...
+2. Reference implementations of PEPs (e.g. `mypy <https://github.com/python/mypy>`_)
+3. Tooling and support around CPython and the language (e.g. `python.org repository <https://github.com/python/pythondotorg>`_)
+4. PSF-related repositories (e.g. the `Code of Conduct <https://github.com/python/pycon-code-of-conduct>`_)
```
Comment (ewdurbin) [inline devcycle.rst]: While this is general enough to cover what I mentioned, I might suggest:

```suggestion
4. PSF-related repositories (e.g. the `Code of Conduct <https://github.com/python/pycon-code-of-conduct>`_)
5. PSF Infrastructure repositories (e.g. the `PSF Infrastructure Salt configurations <https://github.com/python/psf-salt>`_)
```

**Pull request merged**